### PR TITLE
env files added and readme updated

### PR DESCRIPTION
**Pull request by JRDN-13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,8 @@
-# Northcoders House of Games API
+In order to successfully conenct to the two databases locally you will need to add the following files;
 
-## Background
+.env-development
+.env-test
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+Within each file you will need to add the relevant database name ie PGDATABASE=<databasename>
```
Comment (HannahBurks) [inline README.md]: If you want someone to be able to use the repo (such as a potential employer , you will need to specify the names of each of the databases you have available)

File: db/data/test-data/index.js [hunk 2]
```diff
@@ -2,3 +2,5 @@ exports.categoryData = require('./categories.js');
 exports.commentData = require('./comments.js');
 exports.reviewData = require('./reviews.js');
 exports.userData = require('./users.js');
+
+module.exports = { categoryData, reviewData, userData, commentData}
```
Comment (HannahBurks) [inline db/data/test-data/index.js]: We do not need this module.exports and exports. is another way of doing this :)

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Please revert your test data file and add additional details to your readme 😄

File: db/data/test-data/index.js (hunk 2 above)
Comment (JRDN-13) [reply db/data/test-data/index.js]: Bit confused what I need to do here as the ticket said to export on a object. Would it be something like 

exports.indexData = { categoryData
reviewData
userData
commentData
}

File: README.md (hunk 1 above)
Comment (JRDN-13) [reply README.md]: readme updated

File: db/data/test-data/index.js (hunk 2 above)
Comment (two-triangles) [inline db/data/test-data/index.js]: please remove this as it is already done by default using the dot notation :) - there is no need to amend this file

**Pull request merged**